docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -285,7 +285,7 @@ spring:
         client-id: ${SERVICE_PRINCIPAL_ID}
```

Comment:
Please help to update "${SERVICE_PRINCIPAL_ID}" to "${AZURE_CLIENT_ID}" thanks, same for "${SERVICE_PRINCIPAL_SECRET}"